src/software/ai/hl/stp/tactic/tactic_test.cpp

```diff
@@ -170,9 +173,14 @@ TEST_P(TacticAvoidAreasTest, test_default_avoid_areas)
         game_state.updateBall(ball);
     }
 
-    auto next_intent = tactic.getNextIntent(game_state);
+    auto next_intent = tactic.getNextIntent();
     ASSERT_TRUE(next_intent);
 
+    MotionConstraintManager motion_constraint_manager;
+    auto motion_constraints =
+        motion_constraint_manager.getMotionConstraints(game_state, tactic);
+    next_intent->setMotionConstraints(motion_constraints);
```

Comment:
Do we actually need to put the `MotionConstraint`'s in intents to test them?

Reply:
Yes if we want to to test that setMotionConstraints works

Reply:
Ah, could we make the testing of that function a separate test? I think that's out-of-scope for this specific test.

Reply:
done

Reply:
setMotionConstraints is actually tested in intenttest